app/services/account_reset_health_checker.rb

```diff
@@ -0,0 +1,35 @@
+module AccountResetHealthChecker
+  module_function
+
+  Summary = Struct.new(:healthy, :result) do
+    def as_json(*args)
+      to_h.as_json(*args)
+    end
+
+    alias_method :healthy?, :healthy
+  end
+
+  # @return [Summary]
+  def check
+    rec = find_request_not_serviced_within_26_hours
+    Summary.new(rec.nil?, rec)
+  end
+
+  # @api private
+  def find_request_not_serviced_within_26_hours
+    records = AccountResetRequest.where(
+      sql, tvalue: Time.zone.now - Figaro.env.account_reset_wait_period_days.to_i.days - 2.hours
+    ).order('requested_at ASC').limit(1)
```

Comment:
Style: this works just fine since we only do this once in a while. Another pattern I've seen is to `select 1` so we aren't sending a lot of data over the network. We only care about existence.